### ISSUE TITLE
Use firefox as package.json name

### ISSUE THREAD

**Issue by tofumatt** [mozilla/node-firefox-cli#1]
I did some inspection of @lmorchard's branch and saw the name was `node-firefox-cli`.

I think we should use the `firefox` npm name/CLI for this, instead of having to type `node-firefox` or `node-firefox-cli`. Thoughts?

**Comment by lmorchard** [comment]
Yeah, the `firefox-cli` binary name is terrible. But, just plain `firefox` will collide with Firefox itself, especially on Linux. You can usually launch the browser from the command line that way.

**Comment by tofumatt** [comment]
Oh, right. Do we want to stick with just `node-firefox` then?

**Comment by lmorchard** [comment]
I also don't like throwing `node-` into the name, because you could use the tool without ever knowing it was implemented in node. Bleah. Names are hard. But, maybe `node-firefox` is less terrible than `firefox-cli`
